src/axolotl/datasets_work_queue.py

```
@@ -0,0 +1,382 @@
+            """Worker process that continuously pulls from work queue."""
+            try:
+                while True:
+                    try:
+                        # Get work with timeout
+                        idx, example = work_queue.get(timeout=1)
+
+                        # Tokenize the example
+                        try:
+                            tokenized = self.prompt_tokenizer.tokenize_prompt(example)
+                            result_queue.put((idx, tokenized, None))
+                        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)